docs/courses/README.md

```diff
@@ -1,28 +1,101 @@
-# Contributing to LibreLingo courses
+# LibreLingo Course Documentation
 
-LibreLingo courses consist of files that contain course content. These files
-are organized in a specific directory structure.
-[Learn how to edit or create courses](editing-courses.md)!
+This document describes how the course system is laid out on LibreLingo and information on how to contribute to the course data.
 
+This document is the first step whether you want to edit an existing course, contribute a translation to an existing course or create a new one from scratch. This is also a good resource to read if you just want to learn about the course system in general.
+
+---
+
+**NOTE**: If you are not clear about anything on this document feel free to post on [Discussions](https://github.com/kantord/LibreLingo/discussions) so that the community can help you further.
```

Comment:
I like the wording of it. Makes it friendlier. But I think there are a couple of things.
- Recently link text is usually kept short. It's a trend or something, but it's common and getting shorter everyday. People link only one word these days. :) One word seems too short, but we can probably make it shorter.
- "Confused?" in all caps may be ok with bold and normal case instead, without a colon (:).
- We can use "Discussions" instead of "Discussion"

Here is my suggestion:
```
    **Confused?**
Ask people in the [Discussions tab on GitHub](https://github.com/kantord/LibreLingo/discussions)!
```

Reply:
> Recently link text is usually kept short. It's a trend or something, but it's common and getting shorter everyday. People link only one word these days. :) One word seems too short, but we can probably make it shorter.

I haven't heard about this/noticed it. Why is that a trend? What's the value given by following this trend? Do you have some article on it?

Reply:
> "Confused?" in all caps may be ok with bold and normal case instead, without a colon (:).

Sure, that makes sense

Reply:
> We can use "Discussions" instead of "Discussion"

Definitely!

Reply:
> > Recently link text is usually kept short. It's a trend or something, but it's common and getting shorter everyday. People link only one word these days. :) One word seems too short, but we can probably make it shorter.
> 
> I haven't heard about this/noticed it. Why is that a trend? What's the value given by following this trend? Do you have some article on it?

Well, I can't explain it that much. I just saw it around. I guess it's quicker to create and people are lazy. It's hard to find articles on a trend, but I found this [accessibility related article](https://www.mtu.edu/accessibility/training/web/link-text/) which has great examples. And it has:
> Linked text should make sense **out of context**. This is because screen readers often navigation from link to link, skipping the surrounding text. They will read each link out loud, in sequential order.
> ...
> - Keep linked phrases as short as possible, while still being meaningful out of context. This will save screen reader users time, as less will need to be spoken. For example,
> 
>    -  "make a gift" is preferable to "to make a gift online please click here"
>    -  "contact us" is preferable to "you can contact us by"

I have to say people are too lazy and it can make it less accessible which we don't want, at least not for docs. Probably a middle ground would be best. I think we can shorten the link text as much as possible to an extent that it gives context.

Having that said, I didn't know about this earlier. So feel free to point out any other "click here" type links.

Reply:
Yeah, keeping it short makes a lot of sense.

I'm not sure if it's really too long:

> [Ask people in the Discussion tab on GitHub]

Perhaps we could re-phrase it this way:

> [Ask people on GitHub Discussions]

Or just

> [Ask people] on GitHub discussions

Reply:
I think the whole line as a link wouldn't look great. And also would be bad for accessibility as the article suggests.

> [Ask people] on GitHub discussions

I think this one is better. So ultimately it becomes something like this:

```suggestion
**Confused?**
[Ask people](https://github.com/kantord/LibreLingo/discussions) on GitHub Discussions.
```

I actually wanted to make this as a notice to the readers that if anything on the page is not clear, they can ask about it on discussions. So help is always there. And this notice is there on every .md page, to help people get to it faster no matter which page they are on.

I wrote things, so I will obviously be biased towards what I wrote. :) But what do you think? As a reader, do you think it communicates the idea? If it is clear, I can make the change.

Reply:
> [Ask people] on GitHub discussions

I like this. I personally can't decide what's the ideal tradeoff between brevity and clarity, but your reasoning seems pretty logical.

Maybe we can create a separate issue to research this in detail, but for this pull request I'll say this it's perfect!